src/main/kotlin/com/forgerock/sapi/gateway/ob/uk/framework/consent/payment/v4/DefaultOBWriteInternationalScheduledConsent5Factory.kt

```diff
@@ -0,0 +1,29 @@
+package com.forgerock.sapi.gateway.ob.uk.framework.consent.payment.v4
```

Comment:
Missing copyright?

Reply:
No source files in the functional tests suite have copyright headers. I believe it is because it was never intended to be released, it's more of an internal tool to help making sure that the new functionalities added to other components are not broken by the latest changes.